src/iks/iks.py

```diff
@@ -126,8 +126,14 @@ def iks(args):
     
     UserInfo = handle_args(args)
     if UserInfo.delete and not UserInfo.terraforming:
+        '''
         delete_dns = DeleteDNS(UserInfo.crn, UserInfo.zone_id, UserInfo.api_endpoint, UserInfo.cis_domain)
         delete_dns.delete_dns()
+        '''
```

Comment:
We shouldn't comment out the DNS record deletion. Our delete function should still be able to delete the DNS records